packages/ui/src/components/shared/settings/ChangeWalletPasswordDialog/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { Breadcrumbs, DialogContent, Link, Typography } from '@mui/material';
+import DialogTitle from 'components/shared/DialogTitle';
+import ChangingWalletPassword from 'components/shared/settings/ChangeWalletPasswordDialog/ChangingWalletPassword';
+import VerifyingPassword from 'components/shared/settings/VerifyingPassword';
+import { settingsDialogActions } from 'redux/slices/settings-dialog';
+import { RootState } from 'redux/store';
+import { Props } from 'types';
+
+interface ChangeWalletPasswordDialogProps extends Props {
+  onClose: () => void;
+}
+
+const ChangeWalletPasswordDialog: FC<ChangeWalletPasswordDialogProps> = ({ onClose }) => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { verifiedPassword } = useSelector((state: RootState) => state.settingsDialog);
+
+  return (
+    <>
+      <DialogTitle onClose={onClose}>
+        <Breadcrumbs>
+          <Link
+            className='cursor-pointer'
+            underline='hover'
+            color='inherit'
+            variant='h6'
+            onClick={() => dispatch(settingsDialogActions.resetState())}>
+            {t<string>('Settings')}
+          </Link>
+          <Typography color='text.primary' variant='h6'>
```

Comment:
`color='text.primary'` is this to set the color to `primary` color -> I don't think this would work, it not then we should remove it.

Reply:
![image](https://user-images.githubusercontent.com/109079568/231610522-6b6ee412-4665-4b5a-a9a2-d1fa709a09b6.png)
We need that for the different color between the `Settings` and `Changing wallet password`. The `primary` is set to blue and so I set it to `text.primary` for black.